specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/examples/MyWorkbookGet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2015-05-01",
+    "subscriptionId": "6b643656-33eb-422f-aee8-3ac145d124af",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "c0deea5e-3344-40f2-96f8-6f8e1c3b5722",
+        "name": "deadb33f-8bee-4d3b-a059-9be8dac93960",
+        "type": "",
+        "location": "westus",
+        "properties": {
+          "name": "My New Workbook",
```

Comment:
can you call this display Name. top level property cant be repeated inside the properties envelope.

Reply:
can't rename /change things, these are all existing apis that just weren't properly documented in swagger.  the whole /myworkbooks type is effectively a copy of the /workbooks type (they were even deployed at the same time, not sure why the swagger stuff wasn't done at the same time)

Reply:
actually now that i look at this, it seems like this is out of date and it IS displayName in the real data.  i'm not sure why the workbooks stuff i copied has it wrong too. :/

Reply:
actually this one IS displayname in the actual place, not sure why the workbooks swagger doesn't have that, so i can fix it here